config/locales/administrate.zh-TW.yml

```diff
@@ -0,0 +1,23 @@
+---
+zh-TW:
+  administrate:
+    actions:
+      confirm: 你確定？
+      destroy: 刪除
+      edit: 編輯
+      show: 檢視
+    controller:
+      create:
+        success: "已成功新增 %{resource}。"
+      destroy:
+        success: "已成功刪除 %{resource}。"
+      update:
+        success: "已成功更新 %{resource}。"
+    fields:
+      has_many:
+        more: 顯示 %{total_count} 筆中的 %{count} 筆資料
```

Comment:
At this URL it's showing up under the "Line Items" table: https://administrate-prototype.herokuapp.com/admin/orders/291

<img width="821" alt="order__291___administrate_prototype" src="https://cloud.githubusercontent.com/assets/829576/11071732/30a57a7c-8797-11e5-851c-1835fe593f8a.png">